python/lsst/cell_coadds/test_utils.py

```diff
@@ -101,8 +101,8 @@ def generate_data_id(
         skymap=skymap_record,
         physical_filter=physical_filter_record,
     )
-    data_id = DataCoordinate.standardize(record, universe=universe)  # type: ignore
-    return data_id.expanded(record)  # type: ignore
+    data_id = DataCoordinate.standardize(record, universe=universe)
+    return data_id.expanded(record)
```

Comment:
There actually is a problem here, though I think it's the kind mypy can't catch, and it may not matter for test utilities - we need two different dicts here:
- one to pass to `DataCoordinate.standardize` with just data ID values
- one to pass to `DataCoordinate.expanded` with just records